src/components/phaseAccordions/PhaseAccordions.tsx

```diff
@@ -0,0 +1,74 @@
+import moment from 'moment'
+import { Accordion, ContentBlock, TaskContentIcon } from '@/components/index'
+import type { AccordionState } from '@/hook/useAccordion'
+import type { Status, TaskDTO, TaskContent } from '@/data/phase/dto.type'
+
+type PhaseAccordionProps = Readonly<{
+  activeAccordion: AccordionState
+  onClick: (taskName: string) => () => void
+  number: number
+  phaseName: string
+  status: Status
+  tasks: TaskDTO[]
+}>
+
+export const formatString = (text: string): string => text.trim().toLowerCase()
```

Comment:
Not meaningful. I would name the function: `normalizeString`, which reflects the specific string transformation applied.